lighthouse-cli/test/smokehouse/test-definitions/forms/form-config.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/**
+ * @type {LH.Config.Json}
+ * Config file for running form gatherer  tests.
+ */
+const config = {
+  extends: 'lighthouse:default',
```

Comment:
note to smoketest authors: in the future we could do something like this:

```
  extends: 'lighthouse:default',
  settings: {
    onlyAudits: [
      'some-form-audit' // Run a single audit that needs the forms artifact.
    ],
  },
```

this would result in the smoketest just gathering from 1 gatherer and doing 1 pass. instead of the default which is ~30 gatherers and 3 passes.

but right now it wont work, as the gatherer will be filtered out of the default-config and no audit actually requires it.

but the filtering doesn't happen if there's no onlyAudits (thus why the smoketest works as is)

https://github.com/GoogleChrome/lighthouse/blob/488c7ba491d8d9acc5d4bde71cb82b6acf62f74c/lighthouse-core/config/config.js#L559-L566

(an alternative is we add something like `onlyArtifacts` support to the config. could be a terrible idea.)

Reply:
https://github.com/GoogleChrome/lighthouse/issues/8931